### ISSUE TITLE
CROWZ (1692070)

### ISSUE THREAD

**Issue by Milliw** [ValveSoftware/Proton#5743]
# Compatibility Report
- Name of the game with compatibility issues: CROWZ
- Steam AppID of the game: 1692070

## System Information
- GPU: RTX 2060
- Driver/LLVM version: nvidia 510.60.02
- Kernel version: 5.16.18
- Link to full system information report as [Gist](https://gist.github.com/MarcoWilli/a9005ebe43d7adab38739a3afa219d4a):
- Proton version: 7.0.1

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms
Game does not start. Seems to me there are hardcoded backslash file paths.
[steam-1692070.zip](https://github.com/ValveSoftware/Proton/files/8404695/steam-1692070.zip)

**Comment by kisak-valve** [comment]
Most likely a general Proton + XIGNCODE3 issue.

**Comment by Milliw** [comment]
So nothing to do with the `CrowFPS\\Binaries\\Win64\\CrowFPSClient-Win64-Shipping.exe` calls via double backslashes?

**Comment by madewokherd** [comment]
There's no reason to expect that to be a problem. It looks like an argument to the game exe which would access the file using the Windows API and thus backslashes would be translated.

**Comment by gofman** [comment]
Yes, the initial problem with unable to connect to server is somewhat related to extra quotes for the process spawned by the game through ShellExecuteW, due to the extra quotes (how that is supposed to work is a separate story) it cannot correctly parse command line and specifically server address.

That is solvable one or another way but then it wants Xigncode3 kernel mode driver. User mode Xigncode works for some games in Proton now, but xigncode with kernel driver looks pretty much as a blocker for now.
